includes/validation/class-amp-validation-manager.php

```diff
@@ -666,7 +666,16 @@ public static function add_validation_error_sourcing() {
 
 		add_filter( 'do_shortcode_tag', [ __CLASS__, 'decorate_shortcode_source' ], PHP_INT_MAX, 2 );
 		add_filter( 'embed_oembed_html', [ __CLASS__, 'decorate_embed_source' ], PHP_INT_MAX, 3 );
-		add_filter( 'the_content', [ __CLASS__, 'add_block_source_comments' ], 8 ); // The do_blocks() function runs at priority 9.
+
+		// The `WP_Block_Type_Registry` class was added in WordPress 5.0.0. Because of that it sometimes caused issues
+		// on the AMP Validated URL screen when on WordPress 4.9.
+		if ( class_exists( 'WP_Block_Type_Registry' ) ) {
```

Comment:
When on WordPress 4.9, I got the following error on AMP Validated URL screen:

<img width="1504" alt="Screenshot 2021-12-08 at 10 05 50" src="https://user-images.githubusercontent.com/478735/145181575-27fce6a0-9cd1-4c92-8f39-5ae9e4f58b2d.png">

This change prevents the error to occur.